### ISSUE TITLE
Cannot restore any history

### ISSUE THREAD

**Issue by hendursaga** [atlas-engineer/nyxt#2425]
With 7f7635ffed2940b51011c14153bfd81b4fda6d64 and running `./nyxt -C -I`, I get this error and Nyxt hangs indefinitely, unless I first delete my history and restart:

```
Warning: Failed to restore history from /home/user/.local/share/nyxt/history/default.lisp: There is no class named COMMON-LISP::HISTORY-ENTRY.
```

**Comment by jmercouris** [comment]
Hm, this is true, no such class exists...

**Comment by bubbleguuum** [comment]
Same issue here. nyxt shows a blank window and must be killed with `kill -9`.

**Comment by hendursaga** [comment]
Also it's giving me...

```
<INFO> [16:04:34] Failed to save #P"/home/user/.local/share/nyxt/history/default.lisp": The symbol "HISTORY-TREE-KEY" is not external in the COMMON-LISP package.
                                                                                          Stream: #<dynamic-extent STRING-INPUT-STREAM (unavailable) from "(:SEQUEN...">
<WARN> [16:04:44] Warning: Error on separate thread: The function COMMON-LISP::HISTORY-TREE-KEY is undefined.
```

**Comment by jmercouris** [comment]
I confirm this issue.

**Comment by Ambrevar** [comment]
Fixed with a0598eddc3f9f0fd3c3251118b28b94fd8740433, sorry for the trouble :p
